app/views/static/privacypolicy.html.erb

```diff
@@ -52,13 +47,13 @@
       We use your Personal Information only for providing and improving the Site. By using the Site, you agree to the collection and use of information in accordance with this policy.
       </p>
 
-      <h3>Information Collection And Use</h3>
+      <h3 class="text-xl font-medium pt-4 pb-2">Information Collection And Use</h3>
 
```

Comment:
I see you are repeating these 3 classes for every section heading.    Can you create a new class for the section headings which uses those 3 classes?    That way, if we change the styling later,  we would only have to change that in one spot instead of seven.

Reply:
It's a tradeoff between the traditional custom CSS approach and the [utility-first inline classes](https://tailwindcss.com/docs/utility-first#overview). If I [extract the duplication out then I need to use a separate CSS file (if I still use tailwind styling lib)](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply), which I think it's more cumbersome just for 1 custom class.